README.md

```diff
@@ -451,31 +466,13 @@ In case the application doesn't work as expected. Here are some things you can v
 
 The `nativescript-push-notifications` module for Android relies on the Firebase Cloud Messaging (FCM) SDK. In the steps below you will be guided to complete a few additional steps to prepare your Android app to receive push notifications from FCM.
 
-1. Add the FCM SDK
-
-> Thе `google-services` plugin is added automatically. If this fails, you can try adding it manually:
-
-- Navigate to the project `platforms/android/` folder and locate the application-level `build.gradle` file
-- Add the `google-services` plugin to the list of other dependencies in your app's `build.gradle` file
-    ```Groovy
-    dependencies {
-        // ...
-        classpath "com.google.gms:google-services:3.0.0"
-        // ...
-    }
-    ```
-- Add the following line be at the bottom of your `build.gradle` file to enable the Gradle plugin
-    ```Groovy
-    apply plugin: 'com.google.gms.google-services'
-    ```
-
 1. Add the `google-services.json` file
 
     To use FCM, you need this file. It contains configurations and credentials for your Firebase project. To obtain this follow the instructions for adding Firebase to your project from the official [documentation](https://firebase.google.com/docs/android/setup). Scroll down to the **Manually add Firebase** section.
 
     Place the file in your app's `App_Resources/Android` folder
 
-1. Obtain the FCM Server Key (optional)
+2. Obtain the FCM Server Key (optional)

```

Comment:
if you want to keep consistent with the rest of the file, ordered lists should be 1 1 1 and not 1 2 3